raiden/tests/scenarios/close_channel_reopen_before_settle.yaml

```diff
@@ -0,0 +1,66 @@
+settings:
+  gas_price: "fast"
+
+token:
+
+nodes:
+  commands:
+    stop: "pkill -f '127.0.0.1:50{:02d}'"
+    kill: "pkill -9 -f '127.0.0.1:50{:02d}'"
+    start: "raiden --config-file raiden_{0:02d}.toml --api-address 127.0.0.1:50{0:02d}"
```

Comment:
I think this means that i have to have a file called this way configured with address & password that are created specifically for this test... the account will differ for each of us and for the tests. How can we resolve this issue?
@ulope any ideas?

Reply:
The address and stuff would be written in the config file that should contain all the info necessary to start the node.